test/StringBuilderLinewiseExtensionsTests.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Text;
+using Microsoft.PowerShell;
+using Xunit;
+
+namespace Test
+{
+    public sealed class StringBuilderLinewiseExtensionsTests
+    {
+        [Fact]
+        public void StringBuilderLinewiseExtensions_LinewiseYank_Fragment()
+        {
+            var buffer = new StringBuilder("line1\nline2");
+
+            // system under test
+
+            var range = buffer.GetRange(1, 42);
+
+            // assert
+
+            Assert.Equal(6, range.Offset);
+            Assert.Equal(5, range.Count);
+        }
+
+        [Fact]
+        public void StringBuilderLinewiseExtensions_LinewiseYank_Line()

```

Comment:
How is this different from `StringBuilderLinewiseExtensions_LinewiseYank_Fragment`?

Reply:
You are right, it is not.
I will keep only one.